host/xtest/pkcs11_1000.c

```diff
@@ -2312,6 +2314,17 @@ static void xtest_pkcs11_test_1011(ADBG_Case_t *c)
 	if (!ADBG_EXPECT_CK_OK(c, rv))
 		goto out;
 
+	/*
```

Comment:
You forgot to move `Do_ADBG_BeginSubCase()`/`Do_ADBG_EndSubCase()`.

Reply:
I added this as a check to ensure all handles are valid. Its not a new Subcase.

Reply:
Ok, then, should remove sub test 9 that is now empty.

Reply:
Sorry for that. Already added a commit for same.